app/views/cieszyn/prices/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :body_class, 'cieszyn' %>
+<div class='container'>
+  <%= render 'cieszyn/bar' %>
+  <h2>Cennik</h2>
+  <% if(@jobs) %>
```

Comment:
We can assume that the price-list will be not empty. Try to avoid the logic in the view.